#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_fix-man-page.dpatch by Arjan Oosting <arjan@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix lintian warning: manpage-has-errors-from-man
## DP: usr/share/man/man1/c2hs.1.gz 109: warning: `Bc2hs' not defined

@DPATCH@
diff -urNad c2hs~/doc/man1/c2hs.1 c2hs/doc/man1/c2hs.1
--- c2hs~/doc/man1/c2hs.1	2008-01-06 23:20:00.000000000 +0100
+++ c2hs/doc/man1/c2hs.1	2008-03-16 14:43:53.000000000 +0100
@@ -106,7 +106,7 @@
 automatically, passing the appropriate flags.
 
 When used directly, 
-.Bc2hs
+.B c2hs
 is usually called as:
 
 .B c2hs
